### ISSUE TITLE
Picajs not working with Brave browsers active protection

### ISSUE THREAD

**Issue by a-tonchev** [nodeca/pica#213]
When I try to use PicaJS on brave it shows following error:

![image](https://user-images.githubusercontent.com/54528540/124604496-a96c8980-de73-11eb-9059-78e07cdf54a5.png)

Some kind protection by brave is not working properly

**Comment by puzrin** [comment]
Error says explicit, your browser prohibits data read from canvas. Here is nothing to fix.

**Comment by a-tonchev** [comment]
Yeah, thanks. Turning off browser (fingerprint) protection solves the problem
